app/build.gradle

```diff
@@ -10,36 +10,37 @@ buildscript {
             rightmesh_artifactory_password=<hash of your developer portal password>
             rightmesh_hellomesh_key=<some key that you have access to>
         */
-        artifactory_app_username = "${rightmesh_artifactory_username}"
-        artifactory_app_password = "${rightmesh_artifactory_password}"
+        artifactory_app_username = "${rightmesh_devportal_username}"
+        artifactory_app_password = "${rightmesh_devportal_password}"
         artifactory_app_key = "${rightmesh_hellomesh_key}"
     }
     repositories {
+        google()
+        jcenter()
+        mavenCentral()
+        maven { url "https://dl.bintray.com/ethereum/maven" }
         maven {
-            url "https://research.rightmesh.io/artifactory/libs-local"
+            url "https://artifactory.rightmesh.io/artifactory/maven"
             credentials {
                 username artifactory_app_username
                 password artifactory_app_password
             }
         }
-        google()
-        mavenCentral()
-        jcenter()
     }
     dependencies {
-        classpath 'io.left.rightmesh:rightmesh-plugin:1.6'
+        classpath 'io.left.rightmesh:rightmesh-plugin:1.8.1'
```

Comment:
would it be better to have our rightmesh block here so that we are consistent all across

Reply:
Updated